hypertrace-view-generator/hypertrace-view-generator/build.gradle.kts

```diff
@@ -34,13 +34,20 @@ dependencies {
   // TODO: migrate in core
   implementation("org.hypertrace.core.viewgenerator:view-generator-framework:0.3.9")
   implementation("org.hypertrace.core.datamodel:data-model:0.1.20")
-  implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.26")
+  implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.33")
 
   implementation("org.hypertrace.entity.service:entity-service-api:0.8.5")
 
   implementation("org.apache.avro:avro:1.10.2")
   implementation("org.apache.commons:commons-lang3:3.12.0")
-  implementation("com.fasterxml.jackson.core:jackson-databind:2.12.2")
+  implementation("com.fasterxml.jackson.core:jackson-databind:2.13.1")
+
+  constraints {
+    implementation("org.apache.logging.log4j:log4j-slf4j-impl:2.17.1") {
```

Comment:
we should update log4j version in `view-generator-framework` repository instead of adding constraint.